app/lib/clients/vault/entity.rb

```diff
@@ -8,6 +8,49 @@ def put_entity(name, policies, metadata = {})
                              metadata: metadata)
       end
 
+      def put_group(name, policies, auth_method = "oidc", metadata = {})
+        Domain.with_advisory_lock(name) do
+          original_policies, original_metadata = get_group_data(name)
+          policies = (policies || []) + original_policies
+          metadata = metadata.merge((original_metadata || {}))
+          client.logical.write("identity/group",
+                               {
+                                 name: name,
+                                 type: "external",
+                                 policies: policies.uniq,
+                                 metadata: metadata
```

Comment:
There are also the "member_entity_ids" and "member_group_ids" fields.  I saw an external group in B++2++ today that had those fields populated.  I'm not sure if those are generated or not.  Probably deserves some investigation.

Reply:
added these as `extra_params` for writing the entity/group